HttplugFactory.php

```diff
@@ -95,11 +93,14 @@ protected function evaluateCondition($condition)
             $evaluatedCondition = true;
 
             // Immediately stop execution if the condition is false
-            for ($i = 0; $i < count($condition) && false !== $evaluatedCondition; ++$i) {
-                $evaluatedCondition &= $this->evaluateCondition($condition[$i]);
```

Comment:
Because `$evaluatedCondition` is checked before the next loop and breaks it if false.